index.html

```diff
@@ -1,10 +1,85 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-	<meta charset="UTF-8">
-	<title>Document</title>
-<link rel="stylesheet" href="/assets/style.css" /></head>
-<body>
-	<h1>Hi</h1>
-</body>
-</html>
\ No newline at end of file
+  <head>
+    <meta charset="UTF-8" />
+    <link rel="stylesheet" href="https://use.typekit.net/yri5dks.css" />
+    <title>Document</title>
+
+    
+  <link rel="stylesheet" href="/assets/style.css" /></head>
+  <body>
+    <div class="start-screen-block">
+      <header>
+        <ul>
+          <li><a href="#">HOME</a></li>
```

Comment:
It's better to use `text-transform: uppercase` to make text uppercase

Reply:
Okay, got it. Done